helm/cluster-resources/templates/allow-all-ingress-from-host-cilium-np.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if eq .Values.cni "cilium" }}
+{{- range $namespace := .Values.cilium.allowAllIngressFromHostIn }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: allow-all-ingress-from-host
+  namespace: {{ $namespace }}
+spec:
+  endpointSelector: {}
+  ingress:
+  - fromEntities:
+    - host
+    - remote-node
```

Comment:
This means nodes (both the localhost and non-localhost) can access pods in the `kube-system` namespace?

Reply:
Allows "host network" ->  "pods in kube-system" ingress traffic.

Reply:
host + remote-node = host network
endpointSelector: {} = all pods